roles/hardening/files/limits.conf

```diff
@@ -0,0 +1,58 @@
+# /etc/security/limits.conf
```

Comment:
Is this a copy pasta ?
is this default file or did we add this line
```
*                hard    core             0
```

Reply:
see https://github.com/ministryofjustice/opg-hardening-formula/blob/master/hardening/templates/limits.conf